scripts/run_e2e_tests.py

```diff
@@ -171,6 +173,7 @@ def cleanup():
 
     for p in processes_to_kill:
         common.kill_processes_based_on_regex(p)
+    build.set_constants_to_default()
```

Comment:
Should we explicitly make the changes here using ` build.modify_constants` that way it will be clear what we are changing it to? (not sure, wdyt?)

Reply:
I mean we are hiding it a bit now but also we can unify both `start` and `run_e2e_tests` this way, since if we were to call `build.modify_constants` here directly we would have to make sure that when the code is changed here it is also changed in `start`.